README.md

```diff
@@ -506,9 +621,18 @@ The `puppetdb::server` class manages the puppetdb server independently of the un
 
 ### puppetdb::master::config
 
-The `puppetdb::master::config` class directs your puppet master to use PuppetDB, which means that this class should be used on your puppet master node. It’ll verify that it can successfully communicate with your puppetdb server, and then configure your master to use PuppetDB.
+The `puppetdb::master::config` class directs your puppet master to use PuppetDB,
+which means that this class should be used on your puppet master node. It’ll
+verify that it can successfully communicate with your puppetdb server, and then
+configure your master to use PuppetDB.
 
-Using this class involves allowing the module to manipulate your puppet configuration files; in particular: puppet.conf and routes.yaml. The puppet.conf changes are supplemental and should not affect any of your existing settings, but the routes.yaml file will be overwritten entirely. If you have an existing routes.yaml file, you will want to take care to use the manage_routes parameter of this class to prevent the module from managing that file, and you’ll need to manage it yourself.
+Using this class involves allowing the module to manipulate your puppet
+configuration files; in particular: puppet.conf and routes.yaml. The puppet.conf
```

Comment:
"Using this class allows the module to manipulate the puppet configuration files puppet.conf and routes.yaml."